nautobot/extras/tests/test_jobs.py

```diff
@@ -24,12 +24,10 @@
 
 
 def get_job_class_and_model(module, name):
-    """Test helper function to look up a job class and job model and ensure the latter is enabled."""
+    """Test helper function to look up a job class and job model."""
     class_path = f"local/{module}/{name}"
     job_class = get_job(class_path)
     job_model = Job.objects.get_for_class_path(class_path)
-    job_model.enabled = True
-    job_model.validated_save()
     return (job_class, job_model)
```

Comment:
I don't think this change is needed personally - I don't mind having both `get_job_class_and_model` and `run_job_for_testing` ensure that the job is enabled.

Reply:
Okay, makes sense - I rolled this function back.